### PR TITLE
chore(tree): remove revert notification

### PR DESCRIPTION
**Pull request by rkrasiuk**
## Description

Remove `Revert` notification since it is never emitted.

### CODE REVIEW

**Comment by rkrasiuk** [comment]
pending review from @rakita

**Review by rakita — APPROVED** [review]
This is made to cover a potential situation where we receive FCU of block that is for example parent of our tip, this would just result of unwinding that blocks from db. So `Revert` is just a subset of `Reorg`

File: crates/storage/provider/src/traits/chain.rs
Comment (mattsse) [inline crates/storage/provider/src/traits/chain.rs:70]: > So Revert is just a subset of Reorg

Can we add this note here?

**Review by mattsse — APPROVED** [review]
> So Revert is just a subset of Reorg

this makes sense, but would like to have this documented

lgtm, pending doc

File: crates/storage/provider/src/traits/chain.rs
Comment (rkrasiuk) [reply crates/storage/provider/src/traits/chain.rs]: done!

**Pull request merged**
